pkg/provider/basic.go

```diff
@@ -165,6 +167,11 @@ func (bo *basicOperations) RefreshToken(ctx context.Context, t *Token, opts ...R
 		return nil, semerr.Map(err)
 	}
 
+	tokenExpiry := cast.ToInt64(o.ProviderOptions["token_expiry"])
```

Comment:
Could we use `strconv.ParseInt` here instead so we don't have to import a third-party library for this?